src/components/form/DatePicker/__tests__/DatePicker.test.tsx

```diff
@@ -0,0 +1,44 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
+import DatePicker from '../DatePicker';
+
+describe('DatePicker Component', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders DatePicker with value', () => {
+    render(<DatePicker label='Value check' value={new Date('01/01/2025')} />);
+    expect(screen.getByLabelText('Value check')).toHaveValue('01/01/2025');
+  });
+
+  it('renders DatePicker with label', () => {
+    render(<DatePicker label='Select Date' />);
+    expect(screen.getByLabelText('Select Date')).toBeInTheDocument();
+  });
+
+  it('checks form values change when DatePicker values are updated', () => {
+    render(
```

Comment:
form here is redundant cause you are not using the form ...
```suggestion
  it('should change DatePicker value', () => {
    render(
```

Reply:
title changed

Reply:
title changed